packages/components/src/components/mobile-drawer/mobile-drawer.scss

```diff
@@ -1,10 +1,38 @@
 /* @define dc-mobile-drawer; weak */
+@keyframes openDrawer {
+    0% {
+        transform: translateX(-100%);
+    }
+    60% {
+        transform: translateX(0);
+    }
+    80% {
+        transform: translateX(-5%);
+    }
+    100% {
+        transform: translateX(0);
+    }
```

Comment:
```suggestion
    60%, 100% {
        transform: translateX(0);
    }
    80% {
        transform: translateX(-5%);
    }
```

Reply:
@maryia-binary this will change the drawer's behavior and it's not working as we expected;

Reply:
@iman-fs it's the same code but shorter😄  We can put 60% and 100% together because they declare the same `transform` property, this is why I suggest it as a shorter version) Please refer to: https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes#when_properties_are_left_out_of_some_keyframes

Reply:
you're right but we can do this as long as we keep the order for our percentages. 
we can do sth like this: 
```
@keyframes identifier {
  0% {
    top: 0;
    left: 0;
  }
  30% {
    top: 50px;
  }
  68%,
  72% {
    left: 50px;
  }
  100% {
    top: 100px;
    left: 100%;
  }
}
```
but we can not use 100% first and then jump to 80%. we need to keep the order

Reply:
@iman-fs the thing is we use it like this in packages/components/src/components/loading/loading.scss line 24, and it's working for me all the time😄